gpflow/optimizers/scipy.py

```diff
@@ -74,12 +75,12 @@ def minimize(
         )
 
     @classmethod
-    def initial_parameters(cls, variables: Variables) -> tf.Tensor:
+    def initial_parameters(cls, variables: Iterable[tf.Variable]) -> tf.Tensor:
```

Comment:
I still think a type alias is the right thing to use in this case. Could you please reuse variables here and in other places?